### DISCUSSION TITLE
How to turn off uyou bottoms

### DISCUSSION THREAD

**Discussion by chmohaa** [PoomSmart/YTABConfig#64] [Q&A]
how do i disable these uyou buttons on the top right of the video player
![DFB7B8B7-0E43-4061-824F-B073C90EBFEB](https://user-images.githubusercontent.com/58805873/221415574-d24c314a-89f3-4b5c-8cde-d152995fbba3.jpeg)

**Comment by arichornlover** [comment]
Go to `uYou Settings > Video Controls Overlay` And then toggle everything off

**Reply by chmohaa** [reply]
I disabled everything, but bottons are still there
